DataFormats/Detectors/EMCAL/include/DataFormatsEMCAL/TriggerRecord.h

```diff
@@ -35,24 +36,28 @@ class TriggerRecord
 
  public:
   TriggerRecord() = default;
-  TriggerRecord(const BCData& bunchcrossing, int firstentry, int nentries) : mBCData(bunchcrossing), mDataRange(firstentry, nentries) {}
+  TriggerRecord(const BCData& bunchcrossing, int firstentry, int nentries) : mBCData(bunchcrossing), mDataRange(firstentry, nentries), mTriggerBits(0) {}
+  TriggerRecord(const BCData& bunchcrossing, uint32_t triggerbits, int firstentry, int nentries) : TriggerRecord(bunchcrossing, firstentry, nentries) { mTriggerBits = triggerbits; }
```

Comment:
It is ok, but you could simply extend existing c-tor to:
````
TriggerRecord(const BCData& bunchcrossing, int firstentry, int nentries, uint32_t triggerbits=0) : mBCData(bunchcrossing), mDataRange(firstentry, nentries), mTriggerBits(triggerbits) {}
````

Reply:
Thanks for merging!

I thought this is cleaner, because the first two entries are event related and the second two determine the data size, so I would like to group them even though the trigger type could be optional.